frontend/app/src/components/history/transactions/TransactionContent.vue

```diff
@@ -68,10 +56,57 @@ const {
   mainPage
 } = toRefs(props);
 
+const userAction = ref(false);
```

Comment:
Do we still need this external one?

Reply:
missed this one... will remove and get from composable

Reply:
used in template:
```html
@input="userAction = true"
```